src/Modules/Admin/Commands.Admin/RotatePowerBIEncryptionKey.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Api.V2.Models;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    public class RotatePowerBIEncryptionKey : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBIEncryptionKey";
+        public const string CmdletVerb = "Rotate";
```

Comment:
Only use approved verbs. You can use rotate as an alias for the cmdlet.
https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands 
I recommend either 'Set', 'Mount', 'Update', 'Deploy', 'Register' but I think Set is the best one

Reply:
Using set and added alias Rotate